sidekick_core/pubspec.yaml

```diff
@@ -10,7 +10,7 @@ environment:
 dependencies:
   args: ^2.1.0
   dartx: '>=0.7.1 <2.0.0'
-  dcli: '>=1.15.0 <1.31.0' # 1.17 requires Dart 2.16 https://github.com/noojee/dcli/pull/192
+  dcli: ^1.15.0
```

Comment:
Need to double check that

Reply:
I remember that there was a problem with the transitive dependency `posix` or `ffi`. But I can't reproduce it. Let's allow the latest version!